.storybook/preview.js

```diff
@@ -30,23 +31,27 @@ addParameters({
     container: ({ children, context }) => (
       <DocsContainer context={context}>
         {children}
-        {<DocFooter />}
+        {<DocFooter feedbackFormLink="https://forms.monday.com/forms/213ebddcb0d423ae5b6178fb6e8f7b3d?r=use1" />}
       </DocsContainer>
     ),
     page: DocsPage,
     components: {
-      h1: ComponentName,
+      h1: ComponentNameDecorator,
       h2: SectionName,
       h3: Title,
       li: AnchorListItem,
       a: LinkComponent,
       p: Paragraph,
       Tip,
+      ComponentName: ComponentNameDecorator,
       ComponentRules,
       UsageGuidelines,
       FunctionArguments,
       FunctionArgument,
-      RelatedComponents
+      RelatedComponents: RelatedComponentsDecorator,
+      Frame,
+      UnstyledList,
+      UnstyledListItem
```

Comment:
what are these?

Reply:
Some storybook components, which we are using...
Putting them here makes it not required to import them in stories files. So, for consistency I've put all of used components here